data/config-schema/src/main/resources/net/corda/schema/configuration/rpc/1.0/corda.rpc.json

```
@@ -48,6 +48,20 @@
           "default": 12000
         }
       }
+    },
+    "websocket": {
+      "description": "Configuration settings for WebSockets.",
+      "type": "object",
+      "default": {},
+      "properties": {
+        "idleTimeoutMs": {
+          "description": "Amount of time in milliseconds for a websocket connection to be idle before timing out and closing the connection.",
+          "type": "integer",
+          "minimum": 0,
```

Shall put something like:
```suggestion
          "minimum": 100,
```
or else value of `0` may be interpreted as "never".